### PR TITLE
102482 conversion target date page

### PR DESCRIPTION
**Pull request by MarkRobinsonDfe**
Create page as per ticket:-
https://dfe-gov-uk.visualstudio.com.mcas.ms/Academies-and-Free-Schools-SIP/_workitems/edit/102482?McasTsid=26110

Dev Test URL is:-
https://localhost:44350/school/application-conversion-target-date?appId=2147483647&applyingSchoolId=96

### CODE REVIEW

File: Dfe.Academies.External.Web/Pages/School/ApplicationConversionTargetDate.cshtml.cs [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+
+			try
+			{
+				//// grab draft application from temp= null
+				var draftConversionApplication = TempDataHelper.GetSerialisedValue<ConversionApplication>(TempDataHelper.DraftConversionApplicationKey, TempData) ?? new ConversionApplication();
+
+				// binding var check
+				var x = TargetDateDifferent;
+				var y = TargetDate;
+				var z = TargetDateExplained;
```
Comment (CathLass) [inline Dfe.Academies.External.Web/Pages/School/ApplicationConversionTargetDate.cshtml.cs:149]: what do these do?

File: Dfe.Academies.External.Web/Pages/School/ApplicationConversionTargetDate.cshtml [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+			</div>
+		</div>
+
+		<br/>
+		<input type="submit" value="Save and return to overview" class="govuk-button govuk-!-margin-top-2 govuk-!-margin-bottom-8"/>
+	</form>
+</div>
+
+@* Existing A2C-SIP code
+	<p class="govuk-body-l">Conversion usually takes around 6 months. It may take longer if the school is part of a private finance initiative (PFI) contract.</p>
```
Comment (CathLass) [inline Dfe.Academies.External.Web/Pages/School/ApplicationConversionTargetDate.cshtml:94]: do you still need this code?

**Review by CathLass — DISMISSED** [review]

File: Dfe.Academies.External.Web/Pages/School/ApplicationConversionTargetDate.cshtml.cs (hunk 1 above)
Comment (MarkRobinsonDfe) [reply Dfe.Academies.External.Web/Pages/School/ApplicationConversionTargetDate.cshtml.cs]: test code while I was debugging. now deleted

File: Dfe.Academies.External.Web/Pages/School/ApplicationConversionTargetDate.cshtml (hunk 2 above)
Comment (MarkRobinsonDfe) [reply Dfe.Academies.External.Web/Pages/School/ApplicationConversionTargetDate.cshtml]: test code while I was debugging. now deleted

**Review by CathLass — DISMISSED** [review]

**Review by CathLass — APPROVED** [review]

**Pull request merged**